packages/from-hardhat/src/config.ts

```diff
@@ -0,0 +1,36 @@
+import type { HardhatConfig } from "hardhat/types";
+
+import Config from "@truffle/config";
+
+export const fromHardhatConfig = (hardhatConfig: HardhatConfig): Config => {
+  return Config.default().merge({
+    networks: Networks.fromHardhatConfig(hardhatConfig)
+  });
+};
+
+namespace Networks {
+  export const fromHardhatConfig = (hardhatConfig: HardhatConfig) =>
+    Object.entries(hardhatConfig.networks)
+      .flatMap(([networkName, networkConfig]) => {
```

Comment:
Oh hey, TIL `flatMap` is a thing. Just pretend that I used it in my example code above :-p

Reply:
`flatMap` is great